docs/reference/ingest/processors/fingerprint.asciidoc

```diff
@@ -0,0 +1,89 @@
+[role="xpack"]
+[testenv="basic"]
+[[fingerprint-processor]]
+=== Fingerprint processor
+++++
+<titleabbrev>Fingerprint</titleabbrev>
+++++
+
+Calculates a hash that can be used for content fingerprinting based on the
+content of the specified fields in a document.
+
+[[fingerprint-options]]
+.Fingerprint Options
+[options="header"]
+|======
+| Name               | Required | Default       | Description
+| `fields`           | yes      | n/a           | Array containing the names of the document
+fields whose content will be used to compute the fingerprint
+| `target_field`     | no       | `fingerprint` | Output field for the fingerprint.
+| `salt`             | no       | <none>        | Salt value that will be applied to every
+fingerprint calculated by this processor
+| `method`           | no       | `SHA-1`       | The cryptographic hash function used to
+compute the fingerprint. Must be one of `MD5`, `SHA-1`, `SHA-256`, or `SHA-512`.
+| `ignore_missing`   | no       | `false`       | If true, any missing elements in `fields`
+will be ignored when computing the fingerprint. If all fields are missing, the processor
+silently exits without modifying the document.
+
+include::common-options.asciidoc[]
+|======
+
+[discrete]
+[[fingerprint-processor-ex]]
+===== Examples
+
+The following example illustrates the use of the fingerprint processor:
+
+[source,console]
+----
+POST _ingest/pipeline/_simulate
+{
+  "pipeline": {
+    "processors": [
+      {
+        "fingerprint": {
+          "fields": ["user"]
+        }
+      }
+    ]
+  },
+  "docs": [
+    {
+      "_source": {
+        "user": {
+          "last_name": "Smith",
+          "first_name": "John",
+          "date_of_birth": "1980-01-15",
+          "is_active": true
+        }
+      }
+    }
+  ]
+}
+----
+
+Which produces the following result:
+
+[source,console-result]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        ...
+        "_source": {
+          "fingerprint" : "WbSUPW4zY1PBPehh2AA/sSxiRjw=",
+          "user" : {
+            "last_name" : "Smith",
+            "first_name" : "John",
+            "date_of_birth" : "1980-01-15",
+            "is_active" : true
+          }
+        }
+      }
+    }
+  ]
+}
+----
+// TESTRESPONSE[s/\.\.\./"_index":"_index","_id":"_id","_ingest":{"timestamp":$body.docs.0.doc._ingest.timestamp},/]
+// NOTCONSOLE
```

Comment:
I tested this snippet locally with the changes in #68415. Should be good to remove `// NOTCONSOLE` once #68415 is merged.